engine/renderer/private/pipelines/ssao_pipeline.cpp

```diff
@@ -0,0 +1,233 @@
+﻿#include "pipelines/ssao_pipeline.hpp"
+
+#include "../vulkan_helper.hpp"
```

Comment:
Why go a directory back? You can just include it without doing so

Reply:
My bad, clion auto-complete probably incldued it like this